doc/content/verification_and_validation/ver-1fd.md

```diff
@@ -0,0 +1,35 @@
+# ver-1fd
+
+# Convective Heating
+
+## Case set up and analytical solution
+
+The fourth heat transfer problem (after [ver-1fa](ver-1fa.md), [ver-1fb](ver-1fb.md), and [ver-1fc](ver-1fc.md)) models focuses on the heating of a semi-infinite slab by convection at the boundary. This case is extracted from TMAP7 [!cite](ambrosek2008verification).
+The slab is initially configured with a constant temperature of 100 K throughout the slab. A convection boundary is activated at the surface from time $t = 0$ s. The analytical solution is provided by [!cite](Incropera2002):
+
+\begin{equation} \label{eq:analytical_solution}
+T(x,t) = T_i + (T_{\infty}-T_i) \left[ \text{erfc}\left( \frac{x}{2 \sqrt{t \alpha}} \right)-\exp\left( \frac{hx}{k} + \frac{h^2t \alpha} {k^2}\right) \text{erfc}\left( \frac{x}{2 \sqrt{t \alpha}} + \frac{h \sqrt{t \alpha}}{k} \right)\right],
+\end{equation}
+where $T_i = 100$ K is the initial temperature, $T_{\infty} = 500$ K is the temperature of the enclosure, $h = 200$ W/m$^2$/K is the conduction coefficient, $k = 401$ W/m/K is the thermal conductivity, $\text{erfc}$ is the complimentary error function, $x$ is the position in the slab in m, and
+\begin{equation} \label{eq:analytical_solution_alpha}
+\alpha = \frac{k}{\rho C_p}
+\end{equation}
+is the thermal diffusivity. The volumetric specific heat is defined as $\rho C_p = 3.439 \times 10^6$ J/m$^3$/K, which gives us $\alpha \approx 1.17 \times 10^{-4}$ m$^2$/s.
```

Comment:
Since you cited it during #124, you should cite where these numbers came from here as well.

Reply:
In this case, the TMAP7 documentation never calls out any specific material, so the values are just coming from TMAP7 (except for the typo/inconsistencies), but are not directly linked to a material system.